### PR TITLE
slot per account in StorableAccounts

### PR DESCRIPTION
**Pull request by jeffwashington**
#### Summary of Changes

Fixes #

### CODE REVIEW

File: runtime/src/storable_accounts.rs [hunk 1]
```diff
@@ -36,3 +42,91 @@ impl<'a, T: ReadableAccount + Sync> StorableAccounts<'a, T> for (Slot, &'a [(&'a
...
+}
+
+#[cfg(test)]
+pub mod tests {
+    use {
+        super::*,
+        solana_sdk::account::{AccountSharedData, WritableAccount},
+    };
+
+    fn compare<'a, T: ReadableAccount + Sync + PartialEq + std::fmt::Debug>(
```
Comment (brooksprumo) [inline runtime/src/storable_accounts.rs:78]: Does this need to be `Sync` or `Debug` here?

**Review by brooksprumo — DISMISSED** [review]
Looks good to me! Had one question, but does not need to gate this PR.

File: runtime/src/storable_accounts.rs (hunk 1 above)
Comment (jeffwashington) [reply runtime/src/storable_accounts.rs:78]: Debug is required for assert_eq! apparently. Sync may just be left over.

Comment (jeffwashington) [reply runtime/src/storable_accounts.rs:78]: Sync is required because I need StorableAccounts to be Sync for its normal uses.

Comment (jeffwashington) [reply runtime/src/storable_accounts.rs:78]: @brooksprumo

**Pull request merged**